### PR TITLE
refactor: Better button description for view modes

### PR DESCRIPTION
**Pull request by LucaTanks**
## Type of Change

Language File

## What issue does this relate to?

N/A

### What should this PR do?

Change wording of the view modes button to be easily understandable

### What are the acceptance criteria?

**Checks Needed:**
- Spelling
- Translated to other languages
- Button sizing adapts correctly

### CODE REVIEW

File: src/nginxconfig/i18n/en/templates/app.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default {
     title: `${common.nginx}Config`,
     description: `The easiest way to configure a performant, secure, and stable ${common.nginx} server.`,
     singleColumnMode: 'Single column mode',
-    splitColumnMode: 'Split column mode',
+    splitColumnMode: 'Side by side mode',
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/en/templates/app.js:33]:
```suggestion
    splitColumnMode: 'Side-by-side mode',
```
'Side-by-side' should be hyphenated, please

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

**Review by MattIPv4 — APPROVED** [review]
LGTM, ty

**Comment by MattIPv4** [comment]
Hey, @LucaTanks - Thanks a ton for this PR! 😄

Would you please shoot me an email when you get a chance?
mcowley at digitalocean dot com 🎉

**Comment by LucaTanks** [comment]
Hi, could you add the `hacktoberfest-accepted` label?

**Comment by MattIPv4** [comment]
This repo is already participating in Hacktoberfest, so you should be good to go as this PR has been merged

**Pull request merged**